acceptance/Gemfile

```diff
@@ -1,11 +1,9 @@
-# Specifies a gem mirror; duplicated in acceptance setup
-# to ensure a similar environment on acceptance hosts.
-source ENV['GEM_SOURCE'] || 'https://artifactory.delivery.puppetlabs.net/artifactory/api/gems/rubygems/'
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
```

Comment:
What was the problem with defaulting to artifactory?

Reply:
Definitely not a necessary change, but it's a convention that we follow in most of the other Puppet repositories I've seen (Puppet, Facter, Hiera). I assume the original intent was that you can set `GEM_SOURCE` in CI for Artifactory, but if you're outside of the Puppet internal network you can fall back to rubygems.org, which should be available anywhere.